marketplacetests/payment/app.py

```diff
@@ -6,8 +6,10 @@
 from marionette.by import By
 from marionette.wait import Wait
 
+from marketplacetests.marketplace_base import MarketplaceBase
 
-class Payment(Base):
+
+class Payment(MarketplaceBase):
```

Comment:
Good point, @davehunt. Although I think that `Payment` could be viewed as separate (i.e., there could be other apps on the phone that require payment capabilities that I _think_ might be able to use the same Payment app), for the purposes of this repo all we are testing is Marketplace, so I think it's fair to make `Payment` extend `Marketplace`. I will do that.